doc/files.md

```diff
@@ -26,15 +26,16 @@ Linux    | `$HOME/.bitcoin/`
 macOS    | `$HOME/Library/Application Support/Bitcoin/`
 Windows  | `%APPDATA%\Bitcoin\` <sup>[\[1\]](#note1)</sup>
 
-2. The non-default data directory path can be specified by `-datadir` option.
+2. The data directory path can be specified by `-datadir` option.
```

Comment:
Since this is a full sentence, I guess it should have s/by/by the/.

Maybe

"A custom data directory path can be specified by the `-datadir` option."

or

"A non-default data directory path can be specified by the `-datadir` option."

Reply:
Thankc! [Updated](https://github.com/bitcoin/bitcoin/pull/20076#issuecomment-703297165).